common/djangoapps/student/helpers.py

```diff
@@ -666,3 +667,28 @@ def do_create_account(form, custom_form=None):
         raise
 
     return user, profile, registration
+
+
+def is_block_user(user):
```

Comment:
Hi @huynguyen-groove ,

Rename is_block_user to is_disabled_user to make consistent feature name